lib/database/repositories/EnvironmentHistoryItemRepository.js

```diff
@@ -34,6 +61,17 @@ class EnvironmentHistoryItemRepository extends Repository {
     async bulkInsert(historyItems) {
         return EnvironmentHistoryItem.bulkCreate(historyItems, { returning: false });
     }
+
+    /**
+     * Returns the occurrences of each environment history combination for a given period.
+     *
+     * @param {Period} period the period on which occurrences must be counted
+     * @return {Promise<{environments: string, count: number}[]>} the resulting histogram
```

Comment:
It is not really environment, but statusHistory

Reply:
You're right, it was the same reason as the comment above.